woocommerce-services.php

```diff
@@ -360,9 +360,17 @@ public function load_textdomain() {
 		 */
 		public function init() {
 			$this->load_dependencies();
+
 			add_action( 'admin_init', array( $this, 'admin_enqueue_scripts' ) );
 			add_action( 'admin_init', array( $this->nux, 'set_up_nux_notices' ) );
 
+			// if the TOS has not been accepted or Jetpack is not Connected do nothing else.
+			$jetpack_status = $this->nux->get_jetpack_install_status();
+			$is_jetpack_connected = WC_Connect_Nux::JETPACK_CONNECTED === $jetpack_status || WC_Connect_Nux::JETPACK_DEV === $jetpack_status;
+			if ( true !== WC_Connect_Options::get_option( 'tos_accepted' ) || ! $is_jetpack_connected ) {
```

Comment:
@nabsul This check is failing for me because `tos_accepted` is coming back as a string `1`. Are we now storing the value differently? Do we need to migrate existing settings or use a non-strict check here?